README.md

```diff
@@ -15,12 +15,14 @@ a collection of utilities that can be used to enable authn/authz flows for a kub
 
 gangway is comprised of a deployment and a service. The service may be exposed in any form you prefer be it Loadbalancer directly to the service or an Ingress.
 
-[auth0 example](examples/auth0-gangway-example.yaml)
+* [auth0 example](examples/auth0-gangway-example.yaml)
+
+* [Google OAuth example](#google-oauth-config)
 
 
 ### Creating Config Secret
 
-The [gangway config](#gangway-config) should be stored as a secret in Kubernetes becaue it contains sensitive information. Create a gangway config based on the example, assuming you named the file `gangway-config.yaml` run the following command: `kubectl create secret generic gangway --from-file=gangway.yaml=gangway-config.yaml`
+The [gangway config](#gangway-config) should be stored as a secret in Kubernetes becaue it contains sensitive information. Create a gangway config based on the example, assuming you named the file `gangway-config.yaml` run the following command: `kubectl create secret generic gangway --from-file=gangway.yaml`
```

Comment:
Is there a reason why we need to change this?

Reply:
thought it was a merge error given the two `=`

Reply:
This is how you can specify the file name as exposed by the Secret.

Reply:
Ah gotcha wasn't aware you could specify key-value through command line like that. Will restore to original value.